sql/queries/swim-trackpoints.sql

```diff
@@ -23,6 +23,7 @@
 
 select L.start_time as timestamp,
        (select max(T.distance) from A_TRACKPOINT T where T.length_id = L.id) as dst,
+       (select round(avg(T.heart_rate)) from A_TRACKPOINT T where T.length_id = L.id) as hr,
```

Comment:
I don't think using `round` here is a good idea -- I think the HR should be kept as a average with the entire precision (your HR is also not an exact number, even if the watch records it as such...).  All the display functions use rounding, so the HR would still be displayed as an integer (if they don't, we should fix those).